src/map/script.cpp

```diff
@@ -18319,6 +18319,10 @@ BUILDIN_FUNC(setunitdata)
 			case UMOB_ADELAY: md->base_status->adelay = (short)value; calc_status = true; break;
 			case UMOB_DMOTION: md->base_status->dmotion = (short)value; calc_status = true; break;
 			case UMOB_TARGETID: {
+				if (value==0) {
+					mob_unlocktarget(md,0);
```

Comment:
is 0 as tick really ok ? @Lemongrass3110, @aleos89 
wouldn't be better to use `gettick()`

Reply:
I would say for this unique case when sending 0 you’re specifically telling the script engine you want to release the target. This would be a custom case though that we can define.

Reply:
compiled both, and both of them works.

Reply:
How much of an offset? Something marginal like <1 second? Also, might be worth adding a note in the script command documentation that sending 0 for these params will release the target!

Reply:
it's less than a second, so i think its fine using `gettick()`
I have added the script commands doc, but by logic if getunitdata umob_targetid return 0 if there is no target then setunitdata should do the same.